### DISCUSSION TITLE
Double arrows in Mermaid diagrams do not render correctly

### DISCUSSION THREAD

**Discussion by wigging** [community/community#109385] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Double arrows `<-->` in Mermaid diagrams do not render properly on GitHub. The size of arrows on each end of the line should be the same. But on GitHub one of the arrows is much smaller than the other.

```mermaid
flowchart LR
    A[HTTP client] <--> B
    subgraph " "
        B[HTTP server] <--> C[CA client]
    end
    C <--> D[CA server]
```

This diagram is from the following code:

```
flowchart LR
    A[HTTP client] <--> B
    subgraph " "
        B[HTTP server] <--> C[CA client]
    end
    C <--> D[CA server]
```

**Comment by gil-son** [comment]
Try to do that:

mermaid
graph LR
  A[HTTP client] -->| | B
  B[HTTP client] -->| | A
  subgraph " "
    B[HTTP server] -->| | C[CA client]
  end
  C -->| | D[CA server]


```mermaid
graph LR
  A[HTTP client] -->| | B
  B[HTTP client] -->| | A
  subgraph " "
    B[HTTP server] -->| | C[CA client]
  end
  C -->| | D[CA server]
```

If I supported you consider to mark my answer

**Reply by wigging** [reply]
I want to use one line with arrows at each end. Not two separate lines with arrows. Double arrows `<-->` work fine in Mermaid except on GitHub.
